060_nineDivisor.js

```diff
@@ -0,0 +1,26 @@
+// 60. Number of integers which has exactly 9 divisors
```

Comment:
Not clear of the logic
9 divisors means a number should have exactly 9 factors. From 0 to 100, there are only 2 numbers 48 and 80 that have exactly 9 factors